projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -936,39 +982,127 @@
+  "ars_nouveau.sounds.test": "测试",
+  "ars_nouveau.gui.settings": "动态光源设置",
+  "ars_nouveau.settings.title": "设置",
+  "ars_nouveau.dynamic_lights.button_on": "动态光源已开启。这将在CPU和内存配置较低的电脑上引起卡顿，并有可能在与类似Optifine的优化模组同时使用时产生未知错误。",
+  "ars_nouveau.dynamic_lights.button_off": "动态光源已关闭。",
+  "block.ars_nouveau.orange_sbed": "橙色魔花床",
+  "block.ars_nouveau.blue_sbed": "蓝色魔花床",
+  "block.ars_nouveau.green_sbed": "绿色魔花床",
+  "block.ars_nouveau.purple_sbed": "紫色魔花床",
+  "block.ars_nouveau.red_sbed": "红色魔花床",
+  "block.ars_nouveau.yellow_sbed": "黄色魔花床",
+  "ars_nouveau.starbuncle.set_bed": "已设置床。",
+  "ars_nouveau.summon_bed": "魔花床",
+  "ars_nouveau.page1.summon_bed": "一种用于装饰的床。使用支配之杖可以将星宝石兽绑定到床上，这样它们闲置时就会在床上休息。",
+  "block.ars_nouveau.scryers_oculus": "预知之眼",
+  "block.ars_nouveau.scryers_crystal": "预知水晶",
+  "item.ars_nouveau.scryer_scroll": "预知卷轴",
+  "ars_nouveau.scryers_oculus.no_pos": "未设定卷轴位置。",
+  "ars_nouveau.scryers_oculus.no_scrolls": "附近没有卷轴。请将一个已绑定的预知卷轴放在附近基座上。",
+  "ars_nouveau.scryer_scroll.bound": "已绑定到 %s。",
+  "tooltip.ars_nouveau.scryer_scroll": "对预知水晶使用便能将卷轴绑定到水晶的位置。",
+  "tooltip.ars_nouveau.dowsing_rod": "使用后能短暂地占卜紫水晶的位置，并使得使用者75格范围内的魔法生物发光。单个占卜杖的使用次数有限。",
+  "ars_nouveau.camera.move": "%1$s%2$s%3$s%4$s - 移动",
+  "ars_nouveau.camera.exit": "%1$s%2$s%3$s%4$s - 退出",
+  "ars_nouveau.tooltip.scryers_oculus": "如果你凝视眼，星宝石兽会凝视回来。",
+  "ars_nouveau.page.scryers_crystal": "可以透过其视角观察事物。右击水晶可以进入视角。对其使用空白羊皮纸会得到绑定的预知卷轴。预知卷轴能让你通过预知之眼远程进入水晶视角。",
+  "ars_nouveau.page.scryer_scroll": "用于存储预知水晶的位置。对预知水晶使用空白羊皮纸即可得到绑定的卷轴。将卷轴放在预知之眼附近的基座上便能通过眼远程访问水晶。命名卷轴后，预知之眼界面会有显示，从而能更容易地分辨不同卷轴。",
+  "ars_nouveau.page.scryers_oculus": "能让你远程访问预知水晶。将预知卷轴放在附近基座上，然后与眼互动，这时会出现界面让你选择要访问的预知水晶。进入视角时预知水晶所在区块必须被加载。",
+  "ars_nouveau.scryer_scroll.craft": "对预知水晶使用空白羊皮纸获得。",
+  "item.ars_nouveau.starbuncle_shades": "星宝石兽墨镜",
+  "tooltip.starbuncle_shades": "对星宝石兽使用可让其变酷。但会导致它们不再拾取物品和采摘魔源浆果。",
+  "ars_nouveau.page.starbuncle_shades": "对星宝石兽使用会让它们不再拾取物品和采摘魔源浆果。潜行对其使用支配之杖能脱下墨镜。",
+  "entity.ars_nouveau.ally_vex": "被召唤的恼鬼",
+  "ars_nouveau.scryers_eye.no_scrolls": "请在附近基座放置预知卷轴。",
+  "ars_nouveau.camera.not_loaded": "水晶方块已被移除，或者区块未加载。",
+  "ars_nouveau.page.support_mod": "支持新生魔艺！",
+  "ars_nouveau.patreon": "加入新生魔艺Patreon可获得特殊Discord角色、添加自定义卷册、得到周边商品，还有更多特权！",
+  "ars_nouveau.patreon_text": "Patreon",
+  "entity.ars_nouveau.summon_skeleton": "被召唤的骷髅",
+  "item.ars_nouveau.shapers_focus": "塑造之核心",
+  "ars_nouveau.shapers_focus.tooltip": "用来创造方块",
+  "ars_nouveau.page1.shapers_focus": "能增强移动、构筑、改变方块类型法术的核心。被诸如发射，重力，吸引，击退等影响的方块将在撞击时造成伤害。伤害值会被法术伤害、方块硬度，以及方块飞行速度影响。此外，影响或构筑方块的法术会将后续法术转移到被影响或构筑的目标上。",
+  "ars_nouveau.shapers_focus.blocks": "方块目标",
+  "ars_nouveau.page2.shapers_focus": "影响或构筑方块的法术会将后续法术转移至被影响或构筑的目标上。比如：冻结->破坏 将会冻结方块然后破坏冻结产生的方块。而在没有核心时“破坏”会直接作用于法术直接命中的方块。会造成目标转移的魔法包括：构筑、冻结、破坏、交换、放置等。修饰这些魔符的“扩散”会使所有受影响的方块都被后续法术影响。",
+  "ars_nouveau.page3.shapers_focus": "会移动方块的魔符会将后续法术转移至被移动的方块。若要实践，可以用：构筑->发射->延迟->击退 向你所看的方向发射一块幽灵方块。转移机制会对所有被移动的方块生效。使用“扩散”魔符能让你一次性操纵多个方块。",

```

```suggestion
  "ars_nouveau.page3.shapers_focus": "会移动方块的魔符会将后续法术转移至被移动的方块。可以试着使用：构筑->发射->延迟->击退 向你所看的方向发射一块幽灵方块。转移机制会对所有被移动的方块生效。使用“扩散”魔符能让你一次性操纵多个方块。",
```
改的不是很好，总之找一个表述包含“你试试这个套路”就行
一开始想改成`例如，使用`的，但是感觉玩家看了之后不会主动用